### PR TITLE
Webhook template icons

### PR DESCRIPTION
**Pull request by herflis**

### CODE REVIEW

File: apps/sensenet/package.json [hunk 1]
```diff
@@ -79,6 +79,8 @@
     "webpack-merge": "^5.7.3"
   },
   "dependencies": {
+    "@iconify-icons/logos": "^1.1.9",
+    "@iconify/react": "^1.1.4",
```
Comment (taki9) [inline apps/sensenet/package.json:83]: yarn.lock should have changed?

File: apps/sensenet/src/components/AddButton.tsx [hunk 2]
```diff
@@ -51,15 +51,15 @@ const useStyles = makeStyles((theme: Theme) => {
     },
     listDropdown: {
       padding: '10px 0 10px 10px',
-      width: '245px',
+      minWidth: '245px',
       maxHeight: '548px',
+      width: 'auto',
```
Comment (taki9) [inline apps/sensenet/src/components/AddButton.tsx:56]: auto is its default value, so it is unnecessary to set

Comment (herflis) [reply apps/sensenet/src/components/AddButton.tsx:56]: @taki9 without setting with auto, the dropdown has a defult with and the items are cut in half

**Review by pusztaienike — COMMENTED** [review]
For me it does not seems good: only gatsby has the right icon
image

This branch has also lint error and it seems that github cannot deploy storybook as well. 
(I think it is related to yarn.lock upgrade ?!)

**Review by pusztaienike — APPROVED** [review]
🌞

**Review by taki9 — APPROVED** [review]

**Pull request merged**